nix/nilexplorer.nix

```diff
@@ -54,7 +56,11 @@ stdenv.mkDerivation rec {
     (cd smart-contracts; pnpm run build)
     (cd niljs; pnpm run build)
 
-    (cd explorer_frontend; pnpm run build)
+    cd explorer_frontend
+    export VITE_PLAYGROUND_VERSION=${revCount}
```

Comment:
I've commented in the chat, but I'll put it here as well. If you do it like this, the explorer will be re-built and re-tested every time new comits are made. This is bad for the purposes of caching. Ideally, the derivation/package should only depend on the source files and not the revision or commit number.

What you can do instead is read the version information from a file, and this file can be written to the deb package at the time deb packages is created.

Reply:
Added `VERSION` file in the root